src/main/java/mate/academy/spring/service/RepeatedPasswordValidator.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.service;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class RepeatedPasswordValidator
+        implements ConstraintValidator<RepeatedPassword, UserRequestDto> {
+    private String password;
+    private String repeatedPassword;
+
+    @Override
+    public void initialize(RepeatedPassword constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(UserRequestDto dto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return dto == null || dto.getPassword() == null || dto.getTwinPassword() == null

```

Comment:
Is it possible that `UserRequestDto` will be `null`?

Reply:
I checked DTO so as not to get NPE. it's better to keep here